etherpad/docker-compose.yml

```
@@ -0,0 +1,67 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: etherpad_app_1
+      APP_PORT: 9001
+      
+  app:
+    image: etherpad/etherpad:2.2.7@sha256:980f227a17569901aaa8f98d8bb6eff51e15c04ad3e2ea0a1692662adbcc2090
+    tty: true
+    stdin_open: true
+    volumes:
+      - ${APP_DATA_DIR}/data/plugins:/opt/etherpad-lite/src/plugin_packages
+      - ${APP_DATA_DIR}/data/etherpad-var:/opt/etherpad-lite/var:rw
+    depends_on:
+      - postgres
+    environment:
+      NODE_ENV: production
+      ADMIN_PASSWORD: umbrel
+      DB_CHARSET: utf8mb4
+      DB_HOST: postgres

```

Only worked properly for me when i changed the container name to the full one.
```suggestion
      DB_HOST: etherpad_postgres_1

```